sbpy/spectroscopy/core.py

```diff
@@ -1046,3 +1047,51 @@ def filt(self, bp, unit='W / (m2 um)', **kwargs):
             fluxd = obs.effstim(_unit)
 
         return wave, fluxd
+
+    def color_index(self, wfb, unit):
```

Comment:
Should we allow additional equivalencies to be accepted in case this class needs to be used for customized magnitude system?

`def color_index(self, wfb, unit, equivalencies=None):`

Reply:
Yes, that is the right approach.

Reply:
I don't have any tests in this PR, so I'd rather address this in #124 .